lib/pos_blockchain.go

```diff
@@ -969,6 +969,43 @@ func (bc *Blockchain) getHighestCommittedBlock() (*BlockNode, int) {
 	return nil, -1
 }
 
+func (bc *Blockchain) GetSafeBlocks() ([]*BlockNode, error) {
+	// First get committed tip.
+	committedTip, idx := bc.getHighestCommittedBlock()
```

Comment:
Just a reminder for another PR. Rename to getCommittedTip for consistency, since that's the term we're using for the uncommitted tip too
```suggestion
	committedTip, idx := bc.getCommittedTip()
```

Reply:
addressed in #791